src/main/java/org/terasology/multiBlock2/system/MultiBlockServerSystem.java

```diff
@@ -297,11 +299,10 @@ private void processLoadedMultiBlockMain(EntityRef mainBlockEntity, MultiBlockMa
         }
     }
 
-    private Region3i getChunkRegion(Vector3i chunkPos) {
-        return Region3i.createFromMinAndSize(
-                new Vector3i(chunkPos.x << ChunkConstants.POWER_X,
-                        chunkPos.y << ChunkConstants.POWER_Y,
-                        chunkPos.z << ChunkConstants.POWER_Z), ChunkConstants.CHUNK_SIZE);
+    private BlockRegion getChunkRegion(Vector3i chunkPos) {
+        return new BlockRegion(chunkPos.x << Chunks.POWER_X,
+                        chunkPos.y << Chunks.POWER_Y,
+                        chunkPos.z << Chunks.POWER_Z).setSize(Chunks.CHUNK_SIZE);
```

Comment:
Is there no utility in `Chunks` that does this? If there is not, should we add it? It's basically a lookup of the block region a specific chunks is spanning, right?

Reply:
Please add a `//TODO: provide this as utility on Chunks?`